src/routes/contact.tsx

```diff
@@ -1,196 +1,196 @@
-                    <input
-                      className="form-control"
-                      type="text"
-                      name="name"
-                      placeholder="Your name"
-                      value={formData.name}
-                      onChange={handleInputChange}
-                    />
-                  </div>
-
-                  <div className="form-group mb-3">
-                    <input
-                      className="form-control"
-                      type="email"
-                      name="email"
-                      placeholder="Enter email"
-                      value={formData.email}
-                      onChange={handleInputChange}
-                    />
-                  </div>
-
-                  <div className="form-group mb-3">
-                    <select
-                      className="form-select"
-                      name="subject"
-                      value={formData.subject}
-                      onChange={handleInputChange}
-                    >
-                      <option value="" disabled>
-                        Select a topic
-                      </option>
-                      <option value="Authors Help">Track Package</option>
-                      <option value="Buyer Help">General Enquiry</option>
-                      <option value="Licenses">Others</option>
-                    </select>
-                  </div>
-
-                  <div className="form-group mb-3">
-                    <textarea
-                      className="form-control"
-                      name="message"
-                      cols={30}
-                      rows={10}
-                      placeholder="Write details"
-                      value={formData.message}
-                      onChange={handleInputChange}
-                    ></textarea>
-                  </div>
-                  <button className="btn btn-primary w-100">Send Now</button>
-                </form>
-              </div>
-            </div>
-          </div>
-        </div>
-
-        <div className="container">
-          {/* Google Maps */}
-          <div className="card">
-            <div className="card-body">
-              <div className="google-maps">
-                <h5 className="mb-3">Our office location</h5>
-                <iframe
-                  className="w-100"
-                  src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d37902.096510377676!2d101.6393079588335!3d3.103387873464772!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x31cc49c701efeae7%3A0xf4d98e5b2f1c287d!2sKuala%20Lumpur%2C%20Federal%20Territory%20of%20Kuala%20Lumpur%2C%20Malaysia!5e0!3m2!1sen!2sbd!4v1591684973931!5m2!1sen!2sbd"
-                  allowFullScreen
-                  aria-hidden="false"
-                  tabIndex={0}
-                ></iframe>
-              </div>
-            </div>
-          </div>
-        </div>
-      </div>
-    </>
```

Comment:
Too many divs, try to use other tags like `section` to give more semantics to your hierarchy. Also, there are some components that can be created to simplify the code here, like a card, section and map component.